torch/csrc/cuda/Module.cpp

```diff
@@ -151,8 +152,19 @@ PyObject* THCPModule_getCurrentStream_wrap(
   THPUtils_assert(
       THPUtils_checkLong(device_index), "invalid argument to getCurrentStream");
   int64_t device = THPUtils_unpackLong(device_index);
-  return PyLong_FromUnsignedLongLong(
-      at::cuda::getCurrentCUDAStream(device).pack());
+  auto stream = at::cuda::getCurrentCUDAStream(device);
+  PyObject* output_tuple = PyTuple_New(3);
+  PyTuple_SetItem(
```

Comment:
nit: as this is tuple construction you can use PyTuple_SET_ITEM. This is better because SetItem returns an int error code you need to check, but SET_ITEM does not.